.github/labels.yml

```diff
@@ -208,12 +208,10 @@
   color: '8D5494'
 - name: 'Product Area: Help'
   color: '8D5494'
-- name: 'Product Area: What’s new'
+- name: "Product Area: What's New"
```

Comment:
can this be single quotes for consistency purposes?

Reply:
Well, the value has a single quote. Are you asking to handle that in a different way?

Reply:
Ah, didn't notice the difference between `’` and `'` 🔬